tests/pytest_tests/unit_tests/test_lib/test_filenames.py

```diff
@@ -0,0 +1,41 @@
+import tempfile
+
+import pytest
+from wandb.sdk.lib.filenames import exclude_wandb_fn, filtered_dir
+
+
+def test_filtered_dir_one_parameter():
+    with tempfile.TemporaryDirectory() as tempdir:
+        with open(tempdir + "/foo.txt", "w") as f:
+            f.write("test")
+        filtered_dir(tempdir, lambda path: True, lambda path: False)
+
+
+def test_filtered_dir_two_parameters():
+    with tempfile.TemporaryDirectory() as tempdir:
+        with open(tempdir + "/foo.txt", "w") as f:
+            f.write("test")
+        filtered_dir(tempdir, lambda path, root: True, lambda path, root: False)
```

Comment:
what are the tests here? just that it doesn't break?

Reply:
these two tests ensure that `filtered_dir` can be called with 1- and 2- param pred fn's for `include_fn` and `exclude_fn`.  without the special handling in that function, these tests throw `TypeError` when passed a 1-parameter pred fn.